platform/docs/docs/platform/extensions/modules/data-source.md

```diff
@@ -172,3 +169,44 @@ extensionManager.updateDataSourceConfiguration( "dicomweb",
   },
 );
 ```
+
+## Merge Data Source
+The built-in merge data source is a useful tool for combining results from multiple data sources.
+Currently, this data source only supports merging at the series level. This means that series from data source 'A'
+and series from data source 'B' will be retrieved under the same study. If the same series exists in both data sources,
+the first series retrieved will be used, and any other duplicated series will be ignored.
```

Comment:
this is ambigious wording, do you mean whoever arrives first?

Reply:
Updated wording, thanks!